packages/google_sign_in/google_sign_in_ios/darwin/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -291,30 +293,21 @@ - (GIDConfiguration *)configurationWithClientIdentifier:(NSString *)runtimeClien
 
 - (void)didSignInForUser:(GIDGoogleUser *)user
       withServerAuthCode:(NSString *_Nullable)serverAuthCode
-              completion:(nonnull void (^)(FSIUserData *_Nullable,
-                                           FlutterError *_Nullable))completion
-                   error:(NSError *)error {
-  if (error != nil) {
-    // Forward all errors and let Dart side decide how to handle.
-    completion(nil, getFlutterError(error));
-  } else {
-    NSURL *photoUrl;
-    if (user.profile.hasImage) {
-      // Placeholder that will be replaced by on the Dart side based on screen size.
-      photoUrl = [user.profile imageURLWithDimension:1337];
-    }
-    NSString *idToken;
-    if (user.idToken) {
-      idToken = user.idToken.tokenString;
-    }
-    completion([FSIUserData makeWithDisplayName:user.profile.name
-                                          email:user.profile.email
-                                         userId:user.userID
-                                       photoUrl:[photoUrl absoluteString]
-                                 serverAuthCode:serverAuthCode
-                                        idToken:idToken],
-               nil);
+              completion:
+                  (nonnull void (^)(FSIUserData *_Nullable, FlutterError *_Nullable))completion {
+  NSURL *photoUrl;
+  if (user.profile.hasImage) {
+    // Placeholder that will be replaced by on the Dart side based on screen size.
+    photoUrl = [user.profile imageURLWithDimension:1337];
   }
+
+  completion([FSIUserData makeWithDisplayName:user.profile.name
+                                        email:user.profile.email
+                                       userId:user.userID
+                                     photoUrl:photoUrl.absoluteString
```

Comment:
Swap to dot-notation because `absoluteString` is a property. This is just a convention, there's no functional difference.
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CocoaFundamentals/CocoaObjects/CocoaObjects.html